site/content/docs/developing/domain.en.md

```diff
@@ -1,5 +1,11 @@
 # Domain
 
+!!!attention
+    Today, a route53 domain name can be associated only while creating the app for the first time.  
+    If you'd like to update your application with a domain ([#3045](https://github.com/aws/copilot-cli/issues/3045)), 
```

Comment:
🤔  Is it weird to link to a customer-opened GH issue in our official docs?

Reply:
I think it's fine 🤷 I wanted to link it so that it can generate more 👍 from users